app/javascript/controllers/autocomplete_controller.js

```diff
@@ -0,0 +1,61 @@
+import { Controller } from 'stimulus'
+import axios from 'axios'
+import { csrfToken, jumpToCreator } from './stimulus_modules'
+import autocomplete from 'autocomplete.js'
+import Mustache from 'mustache'
+
+export default class extends Controller {
+  static targets = ['field', 'suggestionTemplate', 'emptyTemplate']
+
+  connect () {
+    this.ac = autocomplete(this.fieldTarget, { hint: false }, [
+      {
+        source: this.source,
+        debounce: 200,
+        debug: true,
+        ariaLabel: 'creators',
+        templates: {
+          suggestion: (suggestion) => {
+            return Mustache.render(this.suggestionTemplateTarget.innerHTML, suggestion)
+          },
+          empty: () => {
+            return Mustache.render(this.emptyTemplateTarget.innerHTML)
+          }
+        }
+      }
+    ]).on('autocomplete:selected', (event, suggestion, dataset, context) => {
+      this.appendSuggestion(suggestion)
+    })
+  }
+
+  appendSuggestion (suggestion) {
+    suggestion.index = document.getElementsByClassName('nested-fields').length
+    axios({
+      method: 'POST',
+      url: this.data.get('post'),
+      data: suggestion,
+      headers: {
+        'X-CSRF-Token': csrfToken()
+      }
+    })
+      .then(function (response) {
+        document
+          .getElementById('creator_aliases')
```

Comment:
Might be a cool idea to pass the `'creator_aliases'` id in as a data-attribute and really decouple this logic from the html